libcudacxx/test/libcxx/CMakeLists.txt

```diff
@@ -0,0 +1,56 @@
+
+set(LIBCXX_LIT_VARIANT "libcxx" CACHE STRING
+    "Configuration variant to use for LIT.")
+
+set(LIBCXX_TEST_LINKER_FLAGS "" CACHE STRING
+    "Additonal linker flags to pass when compiling the tests")
+
+# <nv/target> is found in the parent project, it is required in all headers now
+set(LIBCXX_TEST_COMPILER_FLAGS
+    "-I${libcudacxx_SOURCE_DIR}/include" CACHE STRING
```

Comment:
Would this include dir need to be removed/changed for any reason? It would be preferable to just use `target_include_directories` or better yet, link against the `libcudacxx::libcudacxx` target unless there's a reason to expose this as a configuration option.